### DISCUSSION TITLE
esLibre 2024 - May 24th & 25th, Valencia, Spain

### DISCUSSION THREAD

**Discussion by raulillo82** [uyuni-project/uyuni#8225] [Show and tell]
# Info
* https://eslib.re/2024/
* CfP: First deadline January 28th 2024 - https://eslib.re/2024/proposals/

# Requirements for the CfP
* Title
* Abstract: max. 1000 chars
* Format:
** Short talk (10-15 minutes)
** Long talk (around 30 minutes)
** Workshop (indicate duration in the description: maximum 2 hours)
* Description: max. 5000 chars
* Name Surname presenter
* Company (optional)
* Email
* Bio: max. 2000 chars
* Other optional fields
** Project website: Page with information about the project (if any).
** Target audience: Who do you think might find this topic interesting (usefulness, related fields, possibilities...)? 
** Personal website: In case you have a personal website you want to share.
** Mastodon (or other libre social networks): Any profile on a free social network is valid, but remember to enter the full URL to know also the instance where you are located so we can find you.
** Twitter: Indicate only your name on Twitter.
** GitLab (or other forge) or general portfolio: Any collaborative coding site where you have projects you would like to share.
** Comments:

# Attendees
* @raulillo82 (provided CfP is approved) , @mmmmmmpc (proposal submitted), +more, add yourself!

# Slides
* To be added

**Comment by raulillo82** [comment]
## Title
Uyuni: the open-source configuration and infrastructure management solution for software-defined infrastructure.
## Abstract
Uyuni is a configuration & infrastructure management tool that saves time, costs and headaches when managing and updating machines, up to the scale of tens of thousands.

Its main focus is about enabling the automated deployment of patches and packages based on software channels and repositories that can be assigned to the systems. However, it is capable of much more!

## Format
- Long talk (around 30 minutes)
- Intermediate (some basic knowledge on the topic is needed and I don't think I can classify the proposal as beginner level)

## Description

Apart from the already described automated deployment of patches and packages, Uyuni makes it easier to onboard and manage any Linux server (not only openSUSE or SUSE Linux Enterprise, but also almost any other Linux flavor, such as CentOS, Rocky Linux, Alma Linux, Ubuntu, Debian, Red Hat Enterprise Linux...), connected to the network, from IoT edge devices to Kubernetes environment, no matter where it is located (private or 3rd party data center or in the public cloud).

Uyuni enables a CLM (Channel Lifecycle Management) feature, allowing administrator to freeze repositories content up to a certain date (which is usually the main filter, but not the only possible one!)

Uyuni is a single tool for automated deployment of hardened OS templates (bare metal/VM/container) to a mass scale of servers and IoT devices for faster, consistent and repeatable provisioning and configuration without compromising speed or security.

The CVE audit feature allows to check the status of public security patches and with OpenSCAP it's also possible to check for specification compliance and apply remediations right from Uyuni.

Uyuni makes use of Salt under the hood for most of the tasks, such as configuration management, package management...

## Presenter
Raúl Osuna Sánchez-Infante

## Email

## Bio

Electrical & Electronics Engineering (Ingeniería de Telecomunicación) in Escuela Técnica Superior de Ingeniería de Bilbao, 2006
Bachelor in Physics by UNED, Spain, 2021.
Working for SUSE since 2013, both onsite in Nuremberg office (Germany, 2013-2018) and remotely since 2018 from his hometown, Vitoria-Gasteiz.
* Technical Support Engineer mainly dedicated to SUSE Manager (Enterprise version of Uyuni), from 2013 to 2023.
* Since September 2023, Software Release Engineer for both Uyuni and SUSE Manager.

Passionate about opensource software.
Amateur chess player and powerlifter.

## Company (optional)
SUSE
## Project website
https://www.uyuni-project.org/

**Comment by juliogonzalez** [comment]
I'd maybe make more explicit that we support a lot of GNU/Linux distributions besides SUSE Linux/openSUSE. Other than that, LGTM.

**Comment by raulillo82** [comment]
Proposal sent for the CfP: https://gitlab.com/eslibre/propuestas/-/merge_requests/447
